src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -55,27 +55,27 @@
         {% set filters = collector.filters %}
         <div class="log-filters">
             <div id="log-filter-type" class="log-filter">
-                <ul class="tab-navigation">
-                    <li class="{{ not has_error_logs and not has_deprecation_logs ? 'active' }}">
+                <ul class="tab-navigation" role="tablist">
```

Comment:
Shouldn't it be a `div` here instead of a `ul` as we don't have `li` elements anymore?

Reply:
You are right! I've just fixed it. Thanks.